modelserver/tasks.py

```diff
@@ -1,6 +1,5 @@
 import logging
 import os
-import pdb
```

Comment:
this change is already on main, you should be able to just rebase and make this go away

Reply:
Ack, will address

Reply:
Done